tools/perf_checker/perf_checker.py

```diff
@@ -0,0 +1,42 @@
+import argparse
+
+import profiler
+
+# The script takes a command and a database path and looks
+# the performance anomalies in the performance history of that
+# command across the profiled runs.
+
+parser = argparse.ArgumentParser()
+parser.add_argument("command", type=str)
+parser.add_argument("db_path", type=str)
+
+args = parser.parse_args()
+
+# Processes the set of previously written logs
+# The threshold (ratio) of allowable performance degradation between profiling runs
+threshold = 1.10
+
+db = profiler.data.FileBasedProfileDB(args.db_path)
+dt = db.find(f"{args.command}")
```

Comment:
Suggest renaming `dt`. It is not clear what the variable contains from its name.